rdrf/rdrf/views/form_view.py

```diff
@@ -487,7 +487,11 @@ def get(self, request, registry_code, form_id, patient_id, context_id=None):
             "registry_form",
             kwargs={"registry_code": registry_code, "patient_id": patient_id, "form_id": form_id, "context_id": context_id}
         ) if context_id != 'add' else ''
-        self.set_code_generator_data(context, empty_stubs=changes_since_version is not None)
+
+        conditional_rendering_disabled = changes_since_version or \
```

Comment:
You should also add this check in the POST handler (line 851).

Reply:
:+1: good point. Initially it was only wanted for "read-only" type testing, but if it's a feature, I will add it to the post as well.